Library/RadarIO.Xamarin.Android/RadarSDKImpl.cs

```diff
@@ -169,6 +169,13 @@ public Task<(RadarStatus, RadarTrip, IEnumerable<RadarEvent>)> StartTrip(RadarTr
             return handler.Task;
         }
 
+        public Task<(RadarStatus, RadarTrip, IEnumerable<RadarEvent>)> StartTrip(RadarTripOptions options, RadarTrackingOptions trackingOptions)
+        {
+            var handler = new TripCallbackHandler();
+            AndroidBinding.Radar.StartTrip(options.ToBinding(), trackingOptions.ToBinding(), handler);
```

Comment:
`trackingOptions` can be null

Reply:
in c# all non-primitive objects can be null. the `ToBinding()` extension method will also handle nulls

Reply:
added null guard